packages/main/src/SegmentedButton.ts

```diff
@@ -96,10 +52,60 @@ const metadata = {
  * @appenddocs sap.ui.webc.main.SegmentedButtonItem
  * @public
  */
+
+@customElement("ui5-segmented-button")
+@languageAware
+/**
+ * Fired when the selected item changes.
+ *
+ * @event sap.ui.webc.main.SegmentedButton#selection-change
+ * @param {HTMLElement} selectedItem the pressed item.
+ * @public
+ */
+@event("selection-change", {
+	detail: {
+		selectedItem: { type: HTMLElement },
+	},
+})
 class SegmentedButton extends UI5Element {
-	static get metadata() {
-		return metadata;
-	}
+	/**
+	 * Defines the accessible ARIA name of the component.
+	 *
+	 * @type {string}
+	 * @defaultvalue: ""
```

Comment:
here defaultValue should be undefined since it's defined in that way on next lines